### PR TITLE
Issue #SB-13854 feat: Validate certificate code post QR code scan implementation

### PR DESCRIPTION
**Pull request by sudip-mukherjee**

### CODE REVIEW

File: src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts [hunk 1]
```diff
@@ -7,12 +13,69 @@ import { Component, OnInit } from '@angular/core';
 export class CertificateDetailsComponent implements OnInit {
   showSuccessModal: boolean;
   loader: boolean;
-  error: boolean;
   viewCertificate: boolean;
-  viewCertificateHtml: boolean;
+  error: boolean;
+  enableVerifyButton = false;
+  certificateCode: any;
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts]: why is this any it should be a string right?

File: src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts [hunk 2]
```diff
@@ -7,12 +13,69 @@ import { Component, OnInit } from '@angular/core';
...
+  wrongCertificateCode = false;
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts]: can you declare this as boolean then assign the false value

File: src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts [hunk 3]
```diff
@@ -7,12 +13,69 @@ import { Component, OnInit } from '@angular/core';
...
+    private sanitizer: DomSanitizer
+  ) { }
 
-  constructor() { }
   ngOnInit() {
+    this.instance = _.upperCase(this.resourceService.instance);
+    this.watchVideoLink = this.sanitizer.bypassSecurityTrustResourceUrl('https://www.youtube.com/embed/2Xgff-gsf0A?rel=0&autoplay=1');
+  }
+
+  verify(event: any) {
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts]: please make the method name as certificateVerify

File: src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts (hunk 1 above)
Comment (sudip-mukherjee) [reply src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts]: done

File: src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts (hunk 3 above)
Comment (sudip-mukherjee) [reply src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts]: done

**Pull request merged**